src/app/[locale]/filter-set.tsx

```diff
@@ -6,13 +6,15 @@ interface Props {
   searchResultFilters: SearchResultFilter[];
   selectedFilters: string[];
   setSelectedFilters: Dispatch<string[]>;
+  'data-test'?: string;
```

Comment:
Perhaps use a generic, explicit name instead, to make clear that it should only be used in a specific context? For example: `attributeValueForE2ETesting`?

Reply:
Not sure what name you mean, is it:

'data-test'?: AttributeValueForE2ETesting;

Or (this will be against my proposal in https://github.com/colonial-heritage/dataset-browser/pull/81#discussion_r1109486971)

attributeValueForE2ETesting: string;  

Or in [line 17](https://github.com/colonial-heritage/dataset-browser/pull/84/files#diff-f8bfee02f5996a2df9269b90e27c1f4d649e6430db0797cba2b4db2d02f36981R17) where I rename the attribute because react cannot handle Kebab Case.

'data-test': attributeValueForE2ETesting

Reply:
We decided offline on the interface name testId. And we will use the HTML attribute data-testid.

I will add this to the documentation.

Reply:
Moved the writing of the documentation to: https://github.com/colonial-heritage/dataset-browser/issues/93